hsm-client/hsm-client

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
```

Comment:
This, and the logrotate and sh file can just be deleted; the HSM client service is managed by the RPM that gets installed.

Reply:
we want the logrotate, but the .sh can indeed just go, it's a placeholder we overwrite in userdata and userdata doesn't need it.

Reply:
As discussed on slack, keeping logrotate as logs still produced